src/mod_auth_token.erl

```diff
@@ -429,22 +425,23 @@ revoke_token_command(Owner) ->
         {error, Error} -> Error
     end.
 
--spec clean_tokens(mongoose_acc:t(), User :: jid:user(), Server :: jid:server()) ->
-          mongoose_acc:t().
-clean_tokens(Acc, User, Server) ->
+-spec clean_tokens(Acc, Params, Extra) -> {ok, Acc} when
+      Acc :: mongoose_acc:t(),
+      Params :: #{jid := jid:jid()},
+      Extra :: map().
+clean_tokens(Acc, #{jid := Owner}, _) ->
     HostType = mongoose_acc:host_type(Acc),
-    Owner = jid:make(User, Server, <<>>),
     try
         mod_auth_token_backend:clean_tokens(HostType, Owner)
     catch
         Class:Reason:Stacktrace ->
             ?LOG_ERROR(#{what => auth_token_clean_tokens_failed,
                          text => <<"Error in clean_tokens backend">>,
-                         user => User, server => Server, acc => Acc,
-                         class => Class, reason => Reason, stacktrace => Stacktrace}),
+                         jid => Owner, acc => Acc, class => Class,
```

Comment:
This code will produce the message like this:
```
> logger:macro_log(#{}, info, #{what => test, text => <<"test">>, reason => test, jid => J}).      
okwhen=2022-11-04T12:08:55.147132+00:00 level=info what=test reason=test pid=<0.8360.0> at=: text=test jid="{jid,<<\"test\">>,<<\"localhost\">>,<<\"mobile\">>}" 
```
I think that would be better to format JID before passing it to logger:
```suggestion
                         jid => jid:to_binary(Owner), acc => Acc, class => Class,
```
And the output:
```
> logger:macro_log(#{}, info, #{what => test, text => <<"test">>, reason => test, jid => jid:to_binary(J)}).
ok
when=2022-11-04T12:10:34.016702+00:00 level=info what=test reason=test pid=<0.8360.0> at=: text=test jid=test@localhost/mobile
```